docker/openemr/5.0.2/openemr.conf

```diff
@@ -69,13 +62,9 @@ CustomLog ${APACHE_LOG_DIR}/access.log combined
 
 
 <VirtualHost _default_:443>
-        #   SSL Engine Switch:
-        #   Enable/Disable SSL for this virtual host.
         SSLEngine on
-
-        SSLCipherSuite "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 \
-                       EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 \
-                       EECDH EDH+aRSA RC4 !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS"
+        SSLHonorCipherOrder on
+        SSLCipherSuite EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH
```

Comment:
Got know idea what above SSL changes to but guessing @danehrlich1 did these for good reasons.

Reply:
Lets just ensure we aren't removing any modern browser compatibilities with this more restricted  SSLCipherSuite

Reply:
Does something like this make more sense:
https://httpd.apache.org/docs/2.4/ssl/ssl_howto.html#onlystrong

Reply:
Some good reading on this topic:
https://stackoverflow.com/questions/28737374/sslciphersuite-aliases
https://blog.qualys.com/ssllabs/2011/10/17/mitigating-the-beast-attack-on-tls